aliases-and-scripts.rst

```diff
@@ -0,0 +1,255 @@
+Aliases and Scripts
+===================
+
```

Comment:
Can you split out aliases-and-scripts into a separate pull  request, please.

Reply:
FWIW the aliases and scripts material seems less universal to me, more your personal setup.  My feeling is that it makes more sense as an appendix compared to the other material.